prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -354,7 +354,7 @@ periodics:
       secret:
         secretName: google-oss-robot-ssh-keys
         defaultMode: 0400
-- cron: "05 17 * * 1-5"  # Bump with label `skip-review`. Run at 10:05 PST (17:05 UTC) Mon-Fri
+- cron: "30 18 * * 1-5"  # Bump with label `skip-review`. Run at 11:30 PST (18:05 UTC) Mon-Fri
```

Comment:
Isn't 1a a better time? Seems like this is more likely to deploy right when someone's going to head off to lunch.

Reply:
good point and done